src/Features/Core/Portable/DisposeAnalysis/DisposeObjectsBeforeLosingScopeDiagnosticAnalyzer.cs

```diff
@@ -103,6 +103,13 @@ protected override void InitializeWorker(AnalysisContext context)
                     ComputeDiagnostics(disposeDataAtExit, notDisposedDiagnostics, mayBeNotDisposedDiagnostics,
                         disposeAnalysisResult, pointsToAnalysisResult);
 
+                    if (disposeAnalysisResult.ControlFlowGraph.OriginalOperation.HasAnyOperationDescendant(o => o.Kind == OperationKind.None))
```

Comment:
could this bail out earlier at the beginning of the try block or does this check depend on the result of ComputeDiagnostics?

Reply:
we can bail out earlier too. However, we will hopefully revert this soon in dev16.3 branch, which is the targeted release for #32100